pkg/aws/cloud_config.go

```diff
@@ -0,0 +1,23 @@
+package aws
+
+import (
+	"github.com/aws/aws-app-mesh-controller-for-k8s/pkg/aws/throttle"
+	"github.com/spf13/pflag"
+)
+
+const (
+	flagAWSRegion      = "aws-region"
+	flagAWSAPIThrottle = "aws-api-throttle"
+)
+
+type CloudConfig struct {
+	// AWS Region for the kubernetes cluster
+	Region string
+	// Throttle settings for aws APIs
+	ThrottleConfig *throttle.ServiceOperationsThrottleConfig
+}
+
+func (cfg *CloudConfig) BindFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&cfg.Region, flagAWSRegion, "", "AWS Region for the kubernetes cluster")
+	fs.Var(cfg.ThrottleConfig, flagAWSAPIThrottle, "throttle settings for aws APIs, format: serviceID1:operationRegex1=rate:burst,serviceID2:operationRegex2=rate:burst")
```

Comment:
Nit, `aws` -> `AWS`

Reply:
the cloud is intended to hold all AWS specific configurations. e.g. both aws services's wrapper and eks specific configuration(if any).

I still prefer cloud, given it's widely used among projects, e.g.: 
1. https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/legacy-cloud-providers/aws/aws.go#L500
2. https://github.com/kubernetes/kops/blob/a9574284465216a26929d8ef5e5fa5ee8e69e92f/upup/pkg/fi/cloudup/awsup/aws_cloud.go
:D

Reply:
updated aws -> AWS :D